### ISSUE TITLE
when i train it with coverage ,the loss is nan when i get 250k iter?

### ISSUE THREAD

**Issue by XuemingQiu** [atulkum/pointer_summarizer#54]
how to solve it ?
thank you!

**Comment by XuemingQiu** [comment]
when you miss the error, please take care of the divide zero in decoder attention computation part , change to fellow:
`normalization_factor = attn_dist_.sum(1, keepdim=True)
import sys
attn_dist = attn_dist_ / (normalization_factor.view(-1,1) + torch.ones_like(normalization_factor.view(-1, 1)) * sys.float_info.epsilon)`
